packages/hint-compat-api/docs/css.md

```diff
@@ -1,84 +1,68 @@
-# Deprecated CSS features (`compat-api/css`)
+# Supported CSS features (`compat-api/css`)
 
 ## What does the hint check?
 
-`compat-api/css` checks if the CSS features used are deprecated in the
-[targeted browsers][browser-context].
+`compat-api/css-next` checks if the CSS features used are
+supported in all [target browsers][browser-context].
 
 ## Why is this important?
 
-Deprecated CSS APIs should not be used as browsers may no longer
-provide support for these APIs. It can be tricky knowing when browser
-support for CSS features have been removed - and whether the support
-was removed for the prefixed or non-prefixed version of the feature.
-This hint will check if you are using features that have been deprecated,
-taking into account prefixes.
+New CSS features are being implemented all the time. It's tricky knowing
+when a feature has become standard among all browsers, when it needs a prefix,
+and when it is not supported. This hint will check if you are using features
+that are not supported by your target browsers, taking into account prefixes
+and feature detection via `@supports` blocks.
 
 ### Examples that **trigger** the hint
 
-The [box-lines][box-lines] property
-was added with the `-webkit-` prefix for Chrome and removed from versions of
-Chrome 67 and onwards.
-Targeted Chrome browsers of versions 67 and up will trigger the hint.
+The [box-flex][box-flex] was
+never added for any version of Internet Explorer. If Internet Explorer is
```

Comment:
Should it be `added to`? I think there's another one later on in case it has to be changed. Also later in the HTML one you use `was never added on any version`.
 
```suggestion
never added to any version of Internet Explorer. If Internet Explorer is
```

Reply:
Sounds better to me - most of this section was pre-existing. I just moved things around a bit.